kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -355,9 +356,9 @@ def foundry_prove(
     br = BugReport(foundry_root / 'bug_report') if bug_report else None
     foundry = Foundry(foundry_root, bug_report=br)
 
-    ag_proofs_dir = foundry.out / 'ag_proofs'
-    if not ag_proofs_dir.exists():
-        ag_proofs_dir.mkdir()
+    save_directory = foundry.out / 'ag_proofs'
```

Comment:
Just a nitpick: rename `'ag_proofs'` to `'apr_proofs'`, or perhaps even to just `'proofs'`.

Reply:
Done.